app-text/aspell/aspell-0.60.6.1.recipe

```diff
@@ -12,9 +12,10 @@ more than one Aspell process is open at once."
 HOMEPAGE="http://aspell.net"
 COPYRIGHT="2000-2006 Kevin Atkinson"
 LICENSE="GNU LGPL v2"
-REVISION="3"
-SOURCE_URI="ftp://ftp.gnu.org/gnu/aspell/aspell-0.60.6.1.tar.gz"
+REVISION="4"
+SOURCE_URI="https://ftp.gnu.org/gnu/aspell/aspell-$portVersion.tar.gz"
```

Comment:
It would be nice to use the mirrors:
```diff
-SOURCE_URI="https://ftp.gnu.org/gnu/aspell/aspell-$portVersion.tar.gz"
+SOURCE_URI="https://ftpmirror.gnu.org/aspell/aspell-$portVersion.tar.gz
+	https://ftp.gnu.org/gnu/aspell/aspell-$portVersion.tar.gz"
```

Reply:
What kind of errors did you get? If your recipe was correct then what you can get is a warning saying that the URI from where it got the tarball is different from what is in the recipe. That happens if you change the `SOURCE_URI` after having downloaded the tarball.
What you probably have right now in `download/aspell-0.60.6.1.tar.gz.uri` is the URI you had when haikuporter downloaded the archive the first time you kicked `hp aspell`.
To get rid of the warning you can delete `download/aspell-0.60.6.1.tar.gz.uri` but this will force haikuporter to download the tarball again.
Another option is to edit that file, `download/aspell-0.60.6.1.tar.gz.uri` in order to replace the `ftp.gnu.org/gnu/` by `ftpmirror.gnu.org/`. Then make sure you have the new two-line `SOURCE_URI` and run `hp -n aspell`. It should not complain.
(Believe me, we have many recipes with two-line `SOURCE_URI`. I can think of `sys-apps/sed/`, `sys-devel/patch/`, for example.)

But maybe you got a real error. In this case this might have been that you forgot to remove the quotes at the end of the first line of the `SOURCE_URI` when you added a second line.

Reply:
You say me to add in SOURCE_URI TWO urls. They are both quoted in one string:

SOURCE_URI="first second"

Haikuporter thinks it is ONE uri first%20second

Reply:
I'm copying it here, without the "\`\`\`diff" ... "\`\`\`" block (to get rid of the `-` and `+`):
```
SOURCE_URI="https://ftpmirror.gnu.org/aspell/aspell-$portVersion.tar.gz
	https://ftp.gnu.org/gnu/aspell/aspell-$portVersion.tar.gz"
```
Is this what you tried?
(BTW, the first character in the second line is a tab, and there are no spaces.)

Reply:
Yes, but i might add \ at the end of the first line. Will recheck later.